app/controllers/sidebar_sections_controller.rb

```diff
@@ -27,6 +27,7 @@ def create
         nil,
         group_ids: SiteSetting.enable_custom_sidebar_sections_map,
       )
+      Discourse.redis.multi { |transaction| transaction.del "site_json" }
```

Comment:
I think a nice improvement here will be to introduce a method in `site.rb` that clears the cache so that the concerns of setting the anon cache stays within the same class.

Also, I don't think we need a Redis transaction here when there is only a single Redis command being executed.

Reply:
Yes, staff can do it

https://github.com/discourse/discourse/blob/main/app/controllers/sidebar_sections_controller.rb#L119

https://github.com/discourse/discourse/blob/main/lib/guardian/sidebar_guardian.rb#L5